LightGBM/data.py

```diff
@@ -124,6 +169,40 @@ def features(df):
         else:
             df[name] = getattr(df["date"].dt, attribute).astype("int16")
 
+    df.dropna(inplace = True)
+    return df
+
+
+def training_data(df):  
```

Comment:
no docs

Reply:
This was an old commit, I added it in a later commit.